pandas/tests/arrays/string_/test_string.py

```diff
@@ -226,6 +226,14 @@ def test_astype_int():
     tm.assert_extension_array_equal(result, expected)
 
 
+def test_astype_float():
```

Comment:
can you use the nulls_fixture (though may have to skip for pd.NaT)

Reply:
Can you expand what for? IIUC `pd.Series(["1.1", null], dtype="string")` should be equal for `dtype` in `pd.NA`, `np.nan`, `pd.NaT` (works), `None`. This behaviour should be tested somewhere else and adding the fixture would just result in  repeat the same test four times.